specs.md

```diff
@@ -191,7 +191,7 @@ BMS-18 | Cell balancing | - The BMS must balance the cells until they are all be
 BMS-19 | Power limits calculation and sending (charge state) | - The BMS must calculate charge power limits based on cell temperatures and SoC to avoid exceeding a cell's defined limits. <br/> - The BMS must send the charge power limits to the charger over CAN at 100Hz or faster.
 BMS-20 | Charger disconnection | Upon sensing charger disconnection, the BMS must throw an AIR shutdown fault and enter the fault state.
 BMS-21 | Entering the charge state | The BMS must only enter the charge state after the init state is complete.
```

Comment:
@kirons96 is this not repeating BMS-16?

Reply:
yeah, but i was just explicit about entering and exiting every possible state